app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java

```diff
@@ -136,11 +131,19 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
+
         // Handle item selection
         switch (item.getItemId()) {
             case R.id.action_display_list:
-                bottomSheetBehaviorForDetails.setState(BottomSheetBehavior.STATE_HIDDEN);
-                bottomSheetBehavior.setState(BottomSheetBehavior.STATE_EXPANDED);
+                if(!opened){
```

Comment:
I would remove "opened" variable at all, since someone else may forgot to set it, and everything can be a mess. Instead you can use bottomSheetBehavior.getState(), to check if it is opened or not.